classes/class-wc-connect-shipping-label.php

```diff
@@ -41,51 +41,60 @@ public function __construct(
 			$this->payment_methods_store = $payment_methods_store;
 		}
 
+		private function get_item_data( WC_Order $order, WC_Order_Item $item ) {
```

Comment:
Are there any situations where `private` is ever needed? We can definitely agree to keep everything public for the purpose of this project, but I don't think it's ideal.

For reference: Standard best practices recommend using private by default to promote well-encapsulate code. Many people would also argue that if the only reason you're making a function public is for testing, then either the class or the test is not well-designed.